dxr/plugins/clang/tests/test_anon_ns/test_anon_ns.py

```diff
@@ -5,13 +5,13 @@ class AnonymousNamespaceTests(DxrInstanceTestCase):
     """Tests for anonymous namespaces"""
 
     def test_function(self):
-        self.found_line_eq('+function:"(anonymous namespace in main.cpp)::foo()"',
+        self.found_line_eq('+function:"<anonymous namespace in main.cpp>::foo()"',
                            'void <b>foo</b>() /* in main */', 5)
-        self.found_line_eq('+function:"(anonymous namespace in main2.cpp)::foo()"',
+        self.found_line_eq('+function:"<anonymous namespace in main2.cpp>::foo()"',
                            'void <b>foo</b>() /* in main2 */', 5)
 
     def test_function_ref(self):
-        self.found_line_eq('+function-ref:"(anonymous namespace in main.cpp)::foo()"',
+        self.found_line_eq('+function-ref:"<anonymous namespace in main.cpp>::foo()"',
                            '<b>foo</b>();  /* calling foo in main */', 12)
-        self.found_line_eq('+function-ref:"(anonymous namespace in main2.cpp)::foo()"',
+        self.found_line_eq('+function-ref:"<anonymous namespace in main2.cpp>::foo()"',
```

Comment:
I suspect the top-level makefile doesn't know how to properly `make clean` the clang plugin. You might have to go into the clang folder and run `make clean`.